contractcourt/chain_watcher.go

```diff
@@ -834,7 +834,7 @@ func (c *chainWatcher) handlePossibleBreach(commitSpend *chainntnfs.SpendDetail,
 
 	// Create an AnchorResolution for the breached state.
 	anchorRes, err := lnwallet.NewAnchorResolution(
-		c.cfg.chanState, commitSpend.SpendingTx,
+		c.cfg.chanState, commitSpend.SpendingTx, nil,
```

Comment:
Think this will cause a nil deference.

Reply:
Correct, but this path isn't exercised in this diff. In part 9, this area is updated to ensure this area is fully updated and passes the relevant unit tests. Part of the motivation with this side feature branch is that we can land incremental PRs that may not 100% implement the desired functionally.

I can update this here, but there's other logic missing that's added in part 9 that would demonstrate no nil panic exists.